### DISCUSSION TITLE
Can not download from website BBC iPlayer

### DISCUSSION THREAD

**Discussion by Hagakui90** [aclap-dev/video-downloadhelper#816] [Help and Q&A]
### Your question

![2024-06-04 05_01_06-BBC iPlayer - CBeebies Bedtime Stories - A-E_ Chris Evans - Even Superheroes Hav](https://github.com/aclap-dev/video-downloadhelper/assets/109181861/67537125-dd55-4614-b023-7f85dfb1bdca)

it always displays Failed download.

### Addon & CoApp version and details

I am using version version: 9.0.2.3
target: google
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"693cac6556b64f68"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "F:\\131. Alex Rider Series - Anthony Horowitz - Ebook + Audio + Video\\Video"
        }
    },
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 3,
    "successfull_dl": 44
}

**Comment by GrampaWildWilly** [comment]
Refer to #520.  You really ought to have found that one before you posted.  A search of this forum with the search key "bbc" would have quickly turned it up.  You should always do a search before you post.
